Resources/ArmTemplates/template.json

```diff
@@ -238,6 +238,9 @@
         },
         "workspaceResourceGroup": {
             "type": "string"
+        },
+        "CmsRedisCacheConnectionString": {
+        "type": "string"
```

Comment:
Please make this 'securestring',  so credentials aren't accidentally leaked via the Azure Portal.

https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/data-types#secure-strings-and-objects for more info.